src/install.c

```diff
@@ -410,7 +410,7 @@ GList* get_install_images(const RaucManifest *manifest, GHashTable *target_group
 			RaucSlot *target_slot = NULL;
 
 			/* Not interested in slots of other classes */
-			if (!g_strcmp0(lookup_image->slotclass, *cls) == 0)
```

Comment:
concerning `g_strcmp0()` personally I find it clearer to have the form *!(strings_equal(stra1, str2))* but I also agree that this is a double inversion when only having `g_strcmp0()` and might confuse individuals that are not me :).